static/js/compose.js

```diff
@@ -13,6 +13,7 @@ var exports = {};
 var user_acknowledged_all_everyone;
 
 exports.all_everyone_warn_threshold = 15;
+exports.announce_warn_threshold = 10;
```

Comment:
We should make this higher than `all_everyone_warn_threshold`, like maybe 60, since we're just sending a message and not mentioning people.

Reply:
OK, I set it to 60 in commit 6af34a1. Originally I had set it to 10 based off of @rishig's suggestion [in this comment](https://github.com/zulip/zulip/pull/7828#issuecomment-352521957) but I see why a higher limit would make sense.